README.md

```diff
@@ -208,7 +197,7 @@ spec:
         lifecycle:
           preStop:
             exec:
-              command: ["/bin/sh", "-c", "systemctl stop kmods-via-containers@simple-kmod"]
+              command: ["/bin/sh", "-c", "modprobe -r simple-kmod"]
```

Comment:
```suggestion
              command: [modprobe, -r, simple-kmod]
```

Reply:
`/bin/sh -c` should already be the container entrypoint right?!

Reply:
I don't think so, the image does not define any entrypoint.

Reply:
This is the default docker entrypoint. See https://stackoverflow.com/questions/21553353/what-is-the-difference-between-cmd-and-entrypoint-in-a-dockerfile

Reply:
In that case, the entrypoint is overridden with `modprobe -r simple-kmod` but in an `exec` call, there is no shell involved.